integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/spatial/DistanceSearchableSortableIT.java

```diff
@@ -62,6 +64,10 @@ public void searchableSortable() {
 
 	@Test
 	public void searchableNotSortable() {
+		assumeFalse(
+				"Skipping test for ES GeoPoint as those would become sortable by default in this case.",
+				TckConfiguration.get().getBackendFeatures().fieldsProjectableByDefault()
+		);
```

Comment:
For this specific test, I'd rather explicitly mark the field as non-projectable, so that we still can check that the Elasticsearch backend behaves correctly for non-sortable fields. Otherwise we wouldn't be testing this behavior at all...

Reply:
I've added a similar test but for a "new" `searchableNotSortableNotProjectable` field, this way it follows that pattern in naming - whatever we set we put in name 😃